packages/amplify_datastore/example/lib/models/Blog.dart

```diff
@@ -23,8 +25,16 @@ import 'package:flutter/foundation.dart';
 class Blog extends Model {
   static const classType = const BlogType();
   final String id;
-  final String name;
-  final List<Post> posts;
+  final String? _name;
+  final List<Post>? _posts;
+
+  String get name {
+    return _name!;
+  }
+
+  List<Post>? get posts {
+    return _posts;
+  }
```

Comment:
Not directly related to this PR -
In previous discussion, we mentioned that for one-to-many relationship, we would return an empty array for the nested field. Does that imply that the `posts` field is always an array type value and is not nullable?

Reply:
Yes we mentioned that but only need to do that for "required" list fields.  If the list is not required and is nullable then we can return a null value no problem.  So no, if the list field is not required we can return null and make it nullable.